sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample2_RecognizeCustomFormsFromFile.cs

```diff
@@ -36,28 +36,36 @@ public async Task RecognizeCustomFormsFromFile()
             string formFilePath = FormRecognizerTestEnvironment.CreatePath("Form_1.jpg");
             string modelId = model.ModelId;
 
-            using (FileStream stream = new FileStream(formFilePath, FileMode.Open))
+            #region Snippet:FormRecognizerRecognizeCustomFormsFromFile
+            //@@ string modelId = "<modelId>";
+            //@@ string formFilePath = "<formFilePath>";
+
+            using var stream = new FileStream(formFilePath, FileMode.Open);
+
+            RecognizeCustomFormsOperation operation = await client.StartRecognizeCustomFormsAsync(modelId, stream);
+            Response<RecognizedFormCollection> operationResponse = await operation.WaitForCompletionAsync();
+            RecognizedFormCollection forms = operationResponse.Value;
+
+            foreach (RecognizedForm form in forms)
             {
-                RecognizedFormCollection forms = await client.StartRecognizeCustomFormsAsync(modelId, stream).WaitForCompletionAsync();
-                foreach (RecognizedForm form in forms)
+                Console.WriteLine($"Form of type: {form.FormType}");
+                Console.WriteLine($"Form was analyzed with model with ID: {form.ModelId}");
+                foreach (FormField field in form.Fields.Values)
                 {
-                    Console.WriteLine($"Form of type: {form.FormType}");
-                    Console.WriteLine($"Form was analyzed with model with ID: {form.ModelId}");
-                    foreach (FormField field in form.Fields.Values)
-                    {
-                        Console.WriteLine($"Field '{field.Name}: ");
+                    Console.WriteLine($"Field '{field.Name}: ");
 
-                        if (field.LabelData != null)
-                        {
-                            Console.WriteLine($"    Label: '{field.LabelData.Text}");
-                        }
-
-                        Console.WriteLine($"    Value: '{field.ValueData.Text}");
-                        Console.WriteLine($"    Confidence: '{field.Confidence}");
+                    if (field.LabelData != null)
+                    {
+                        Console.WriteLine($"  Label: '{field.LabelData.Text}");
                     }
+
+                    Console.WriteLine($"  Value: '{field.ValueData.Text}");
+                    Console.WriteLine($"  Confidence: '{field.Confidence}");
```

Comment:
It seems we forgot to close the single quotes in every output here: `'{...}` => `'{...}'`.

Reply:
Same comment for the URI sample.

Reply:
Updated. Thank you!